css/style.css

```
@@ -1,4 +1,98 @@
 * {
     margin: 0;
     padding: 0;
+}
+
+.menu {
+    height: 110px;
+    width: 100%;
+    background-color: royalblue;
+    display: flex;
+    justify-content: left;
+    align-items: center;
+}
+
+.mainbutton {
+    font-weight: bold;
+    color: white;
+    background-color: royalblue;
+    font-size: 30px;
+    margin: 30px;
+    border: none;
+    font-family: "Arial Black";
+}
+
+.name1 {
+    margin: 25px;
```

try to use even numbers
```suggestion
    margin: 24px;
```